### ISSUE TITLE
TripletLoss

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#289]

**Comment by bhack** [comment]
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/triplet.py

/cc @seanpmorgan @yarri-oss

**Comment by chjort** [comment]
Triplet loss is also implemented in TF Similarity.
https://github.com/tensorflow/similarity/blob/master/tensorflow_similarity/losses/triplet_loss.py

**Comment by LukeWood** [comment]
Great points… triplet in particular this is really tricky!!! This will need some thought.

SimCLR at least the loss still just has two inputs so it’s not as bad

**Comment by LukeWood** [comment]
The KerasCV team is attempting to narrow the package scope in order to more efficiently use our limited resources. As such, we are only accepting contributions directly related to items on the 6 month roadmap for the package.

If you feel this feature could be particularly useful, or would like to contribute it - please feel free to re-open this issue and ping LukeWood, ianstenbit or tanzhenyu

Feel free to read more on the roadmap:
https://github.com/keras-team/keras-cv/blob/master/.github/ROADMAP.md

This is already in tf similarity too.
